File: 240322/씨 오 더블유 2/c-o-w-2.py
n = int(input())
line = input().strip()

count = 0
for i in range(n):
    if line[i] == 'C':
        for j in range(i+1,n):
            if line[j] == 'O':
                for k in range(j+1,n):
                    if line[k] == 'W':
                        count += 1

print(count)

File: 240404/씨 오 더블유 2/c-o-w-2.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    int N, count = 0;
    char line[101];

    scanf("%d",&N);
    scanf("%s",line);
    
    for(int i = 0; i < N; i++)
        if (line[i] == 'C')
            for(int j = i+1; j < N; j++)
                if (line[j] == 'O')
                    for(int k = j+1; k < N; k++)
                        if (line[k] == 'W')
                            count++;

    printf("%d\n",count);
    return 0;
}